Description: Adjust path for hibagent for tests in deeper directory structures
Author: Balint Reczey <balint.reczey@canonical.com>
Forwarded: not-needed
--- a/test/hibagent_test.py
+++ b/test/hibagent_test.py
@@ -8,7 +8,7 @@
 from threading import Thread
 
 import imp
-agent_file = os.path.dirname(os.path.abspath(__file__))+"/../agent/hibagent"
+agent_file = os.path.dirname(os.path.abspath(__file__))+"/../../../../agent/hibagent"
 with open(agent_file) as fl:
     hibagent = imp.load_module('hibagent', fl, agent_file, ('py', 'rb', 1))
 
